### PR TITLE
dns: Add CoreDNS options

### PR DESCRIPTION
**Pull request by chrisohaver**
Add options to enable running the dns perf-tests tests against CoreDNS.

### CODE REVIEW

**Comment by chrisohaver** [comment]
@bowie PTAL

**Comment by chrisohaver** [comment]
@bowei PTAL

**Comment by shyamjvs** [comment]
Will approve once there's lgtm.

File: dns/cluster/coredns-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2016 The Kubernetes Authors.
```
Comment (bowei) [inline dns/cluster/coredns-configmap.yaml]: fix the year (on all files)

File: dns/py/params.py [hunk 2]
```diff
@@ -137,6 +147,25 @@ def set(self, inputs, value):
...
+  def __init__(self):
+    super(CorednsCache, self).__init__('coredns_cache', int)
+
+  def is_relevant(self, attributes):
+    return 'cluster-dns' not in attributes
+
+  def set(self, inputs, value):
+    if value > 0:
+      cf = inputs.configmap_yaml['data']['Corefile']
+      cf = re.sub(r'\n', "\n"
```
Comment (bowei) [inline dns/py/params.py]: Can we not use re.sub of newline character to do this?

**Comment by gmarek** [comment]
cc @miekg

**Comment by chrisohaver** [comment]
@bowei - I updated the copyright dates, but reverted the ones on .py files back, due to failed boilerplate checks in travis.

**Comment by chrisohaver** [comment]
@bowei, ptal

**Pull request merged**